### ISSUE TITLE
Position of popovers contained within button group can break when clicking inside popover.

### ISSUE THREAD

**Issue by linuspahl** [mantinedev/mantine#6992]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.13.3

### What package has an issue?

@mantine/core

### What framework do you use?

create-react-app (CRA)

### In which browsers you can reproduce the issue?

All

### Describe the bug

In my case I have a simple `Button.Group`, which contains a `Menu`. The menu dropdown is not rendered within a portal (`withinPortal={false}`).

![image](https://github.com/user-attachments/assets/25ad54e8-383b-44c3-a56c-bb052c5a8bc1)

When I click on the menu dropdown instead of a menu item, the position of the dropdown changes. The same problem exists for other popovers like the `Popover` component.

![position_problem](https://github.com/user-attachments/assets/6aae3bed-6813-43ab-a5ea-ee2152844557)

The reason seems to be that the button group has the following styling, which effects all focused child elements:

```
.m_80d6d844 :where(*):focus {
    position: relative;
    z-index: 1;
}
```

This overrides the `position: absolute;` of the popover.

### If possible, include a link to a codesandbox with a minimal reproduction

https://codesandbox.io/p/sandbox/mantine-react-template-forked-24mzvw?file=%2Fsrc%2FApp.tsx%3A25%2C1

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by linuspahl** [comment]
A fix could be to define a more specific CSS selector, but I am not sure how the ideal selector should look like.

**Comment by vishal590** [comment]
I cam silver this issue. Can someone assign me?

**Comment by rtivital** [comment]
Fixed in [7.13.5](https://github.com/mantinedev/mantine/releases/tag/7.13.5)
